doc/nrf/device_guides/nrf70/fw_patches_ext_flash.rst

```diff
@@ -126,7 +126,7 @@ Building
 
 See :ref:`nrf7002dk_nrf5340` for general instructions on building.
 
-Additionally, you must enable the ``nrf70-fw-patch-ext-flash`` snippet, set :kconfig:option:`SB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE`, and disable :kconfig:option:`SB_CONFIG_PARTITION_MANAGER`.
+Additionally, you must enable the ``nrf70-fw-patch-ext-flash`` snippet, set ``SB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE``, and disable ``SB_CONFIG_PARTITION_MANAGER``.
```

Comment:
Shouldn't we mention that you can build the sample either with ``nrf70-fw-patch-ext-flash`` snippet or with Partition Manager (``SB_CONFIG_PARTITION_MANAGER``). I am worried that the current wording may be a bit misleading.